css/owned-card.styl

```diff
@@ -58,8 +58,25 @@
       background: lighten(COBOLT_BLUE, 20%)
 
 .collection-card
+  -webkit-border-radius: 1em
```

Comment:
As mentioned, nib will handle this for you, unless there's a specific reason to include this... if so, leave a comment? I usually leave a comment when I do and often it's for `-ms` prefixes for IE solutions.